router/cart.router.js

```diff
@@ -0,0 +1,70 @@
+const express = require("express");
+const router = express.Router();
+const { CartItem } = require("../models/cart.model");
+const { extend } = require("lodash");
+
+router
+  .route("/")
+  .get(async (req, res) => {
+    try {
+      const cartItems = await CartItem.findOne({}).populate(
```

Comment:
You can use find instead of findOne. Tell me if I am wrong?

Reply:
yes both of them will work!